build.gradle

```diff
@@ -53,6 +53,10 @@ subprojects {
     implementation 'org.apache.logging.log4j:log4j-jul'
     implementation "com.lmax:disruptor:${project.'lmax-disruptor.version'}"
 
+    //swagger
+    implementation 'io.springfox:springfox-swagger2:2.9.2'
```

Comment:
Swagger is a poor man's documentation tool We need spring `rest-docs` which is far more superior tool for the API documentation